src/common/constants/support.ts

```diff
@@ -92,6 +92,14 @@ export const EXT_SUPPORT_WEB_LIST: ExtSupportWebsite[] = [
     siteName: 'SCAN',
     logo: 'https://assets.blocksec.com/image/1671777583236-2.png'
   },
+  {
+    name: 'Base',
+    chainID: 8453,
+    chain: 'base',
+    domains: ['basescan.org'],
+    siteName: 'SCAN',
+    logo: 'https://assets.blocksec.com/image/1671777583236-2.png'
```

Comment:
Here's the Optimistic logo placeholder. It should be updated with the Base icon from Blocksec's domain.